hash/risc0/bench/methods/guest/src/bin/sha256_accelerated.rs

```diff
@@ -0,0 +1,21 @@
+
+// accelerated sha2 crate
+#![no_main]
+
+use risc0_zkvm::guest::env;
+use sha2::{Sha256, Digest};
+use risc0_zkvm::guest::env::cycle_count;
+// use base16ct::lower::encode_str;
+risc0_zkvm::guest::entry!(main);
+
+pub fn main() {
+
+    let data: Vec<u8> = env::read();
```

Comment:
If I undestand correctly, risc0 and maybe especially the sha256 acceleration doesn't like to work on bytes. You could try using 32 bit words here instead (of course, 1/4 as many so that the total data length is the same number of bytes)